backend/middlewares/post-middleware.js

```diff
@@ -11,12 +11,12 @@ export const isAuthorMiddleware = async (req, res, next) => {
     }
 
     console.log(post.authorId, userId);
-    if (post.authorId.toString() !== userId) {
+    if (post?.authorId?.toString() !== userId) {
       return res
         .status(HTTP_STATUS.FORBIDDEN)
         .json({ message: RESPONSE_MESSAGES.POSTS.NOT_ALLOWED });
     }
-    next();
+    return next();
   } catch (error) {
```

Comment:
return next()? can we do both in one ? I don't think so it is redundant it should be either return or next()